src/EnergyPlus/ExhaustAirSystemManager.cc

```diff
@@ -695,7 +695,7 @@ namespace ExhaustAirSystemManager {
         Real64 Tin = state.dataZoneTempPredictorCorrector->zoneHeatBalance(thisExhCtrl.ZoneNum).ZT;
         Real64 thisExhCtrlAvailScheVal = ScheduleManager::GetCurrentScheduleValue(state, thisExhCtrl.AvailScheduleNum);
 
-        if (present(FlowRatio)) {
+        if (FlowRatio != 0.0) {
```

Comment:
Is it possible that the calling routine would request a `FlowRatio` of zero? If so, this will behave differently than the optional. Would 1.0 be a better default?

Reply:
> Is it possible that the calling routine would request a `FlowRatio` of zero? If so, this will behave differently than the optional. Would 1.0 be a better default?

@jcyuan2020 Asking again.

Reply:
Yes, that's what I was suggesting.

Reply:
> I think this line should still be kept as 0.0 (or remove this line, no `if` condition at all?)
> 
> Do you mean that for the declaration, the parameter should give a `FlowRatio = 1.0` for default? Something like this? :
> 
> ```
> void CalcZoneHVACExhaustControl(EnergyPlusData &state, int const ZoneHVACExhaustControlNum, Real64 const FlowRatio = 1.0)
> ```

But looking at this again, this line should test for 1.0, not zero.
Previously, if  `FlowRatio` was passed in, then the flows would be ratioed, otherwise it would execute the `else`.
Now, if `FlowRatio` is passed in as 1.0 it will execute the `else` which is incorrect.
Also, if `FlowRatio` is passed in as 0.0 (which if I remember correctly could happen is the central exhaust fan is shut down?), then the `else` is also executed and the flow won't be set to zero.

Reply:
@mjwitte Thanks! It looks that here our goal is to find a mechanism that could flag if the parameter `FlowRatio` is not given during the function call. It then looks that here it should pass in the default of `FlowRatio = -1.0` (or any negative number) and then test for `FlowRate >= 0` to replace the original line of `if (present(FlowRatio)` (or the current `FlowRatio != 0.0`). (?)